core/src/test/java/com/coderanch/blackjack/HandTest.java

```diff
@@ -0,0 +1,69 @@
+package com.coderanch.blackjack;
+
+import com.coderanch.blackjack.Card.Rank;
+import com.coderanch.blackjack.Card.Suit;
+
+import org.junit.experimental.theories.Theories;
+import org.junit.experimental.theories.Theory;
+import org.junit.runner.RunWith;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertThrows;
+
+
+/**
+ * Tests the {@link Hand} class.
+ */
+@RunWith(Theories.class)
+public final class HandTest {
+
+    /**
+     * Tests that {@link Hand#addCard(Card)} maintains the correct score
+     */
+    @Theory
+    public void addCard_maintainsCorrectScore() {
+        var hand = new Hand();
+        hand.addCard(new Card(Rank.ACE, Suit.CLUBS));
+        assertThat(hand.bestScore(), is(equalTo(11)));
+
+        hand.addCard(new Card(Rank.ACE, Suit.CLUBS));
+        assertThat(hand.bestScore(), is(equalTo(12)));
+
+        hand.addCard(new Card(Rank.KING, Suit.CLUBS));
+        assertThat(hand.bestScore(), is(equalTo(12)));
+
+        hand.addCard(new Card(Rank.EIGHT, Suit.CLUBS));
+        assertThat(hand.bestScore(), is(equalTo(20)));
+
+        hand.addCard(new Card(Rank.ACE, Suit.CLUBS));
+        assertThat(hand.bestScore(), is(equalTo(21)));
+    }
```

Comment:
The point of Theories is that you can test a general theory that applies to a wide range of data points. Currently you are testing one specific sequence of values.

I admit this is difficult if you haven't done it before, and it's hard for me to explain in a PR comment. Change the rest of your code first, then I will add a Theory that demonstrates what I mean.